app/ReadmeFormatter.php

```diff
@@ -42,6 +44,11 @@ public function formatHtml($html)
         return '<div class="markdown">'.$formattedHtml.'</div>';
     }
 
+    private function removeAnchors($text)
+    {
+        return preg_replace('/<a .+class=\"anchor\".+>.+<\/a>/i', '', $text);
```

Comment:
Should add some lazy quantifiers here to narrow potential matches. Example: https://regex101.com/r/hNuMQx/1
```suggestion
        return preg_replace('/<a .*?class=\"anchor\".*?>.*?<\/a>/i', '', $text);
```

Reply:
Good thinking and thanks for the suggestions. I've applied them in commit.